public/catalog/v1.0.0/openapi.json

```diff
@@ -22,20 +22,27 @@
   ],
   "tags": [
     {
-      "name": "admins",
```

Comment:
Would be nice to pull out the `['admin', 'users']` block in `/spec/requests` tests

Reply:
@syncrou Yes we should do that as a separate issue